config/deploy.rb

```diff
@@ -124,6 +124,7 @@
           execute :systemctl, '--user', :start, :front_stats_update
           execute :systemctl, '--user', :start, :leader_stats_mainnet_update
           execute :systemctl, '--user', :start, :leader_stats_testnet_update
+          # execute :systemctl, '--user', :start, :leader_stats_pythnet_update
```

Comment:
Should be uncommented.

Reply:
No difference at this point. We're not going to use this task until the code is production.